torchbenchmark/util/experiment/instantiator.py

```diff
@@ -122,8 +122,8 @@ def list_devices() -> List[str]:
     devices = ["cpu"]
     import torch
 
-    if torch.cuda.is_available():
-        devices.append("cuda")
+    if device_type := torch._utils._get_available_device_type():
+        devices.append(device_type)
     return devices
```

Comment:
Hey @guangyey @FFFrog, please take a look at here. I noticed that `torch._utils._get_available_device_type()` has the same behavior as `torch._C._get_accelerator()`. So which one should generally be used?

Reply:
`torch._utils._get_available_device_type()` might be deprecated, so we should use `torch._C._get_accelerator()`, right? But it will be changed in https://github.com/pytorch/pytorch/pull/137493

Reply:
I think you could use `torch._C._get_accelerator` here. I will rebase it in my PR~